sormas-ui/src/main/java/de/symeda/sormas/ui/location/LocationEditForm.java

```diff
@@ -215,9 +214,17 @@ protected void addFields() {
 		facilityTypeGroup.addValueChangeListener(e -> {
 			FieldHelper.removeItems(facility);
 			FieldHelper.updateEnumData(facilityType, FacilityType.getTypes((FacilityTypeGroup) facilityTypeGroup.getValue()));
+			if (facilityTypeGroup.getValue() != null) {
```

Comment:
Can be simplified: `facilityType.setRequired(facilityTypeGroup.getValue() != null);`

Reply:
updated